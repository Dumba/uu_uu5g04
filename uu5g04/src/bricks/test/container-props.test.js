/**
 * Copyright (C) 2019 Unicorn a.s.
 * 
 * This program is free software; you can use it under the terms of the UAF Open License v01 or
 * any later version. The text of the license is available in the file LICENSE or at www.unicorn.com.
 * 
 * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
 * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See LICENSE for more details.
 * 
 * You may contact Unicorn a.s. at address: V Kapslovne 2767/2, Praha 3, Czech Republic or
 * at the email: info@unicorn.com.
 */

import React from "react";
import UU5 from "uu5g04";
import "uu5g04-bricks";

const { mount, shallow, wait } = UU5.Test.Tools;

const CONFIG = {
  mixins: [
    "UU5.Common.BaseMixin",
    "UU5.Common.ElementaryMixin",
    "UU5.Common.NestingLevelMixin",
    "UU5.Common.SectionMixin",
    "UU5.Common.ContentMixin",
    "UU5.Common.LevelMixin",
    "UU5.Common.PureRenderMixin"
  ],
  props: {
    noSpacing: {
      values: [true, false]
    },
  },
  requiredProps: {},
  opt: {
    shallowOpt: {
      disableLifecycleMethods: false
    }
  }
};


describe(`UU5.Bricks.Container`, () => {
  UU5.Test.Tools.testProperties(UU5.Bricks.Container, CONFIG);
});

describe(`UU5.Bricks.Container contains other component`, () => {

  it(`UU5.Bricks.Container containt Paragraph and noSpacing is true -> should render without crash`, () => {
    const wrapper = shallow(
      <UU5.Bricks.Container id={"uuID01"} header="noSpacing={true}" noSpacing>
        <UU5.Bricks.Paragraph id={"uuID02"}/>
     </UU5.Bricks.Container>
    );
    expect(wrapper).toMatchSnapshot();
  });

  it(`UU5.Bricks.Container containt Paragraph and noSpacing is false -> should render without crash`, () => {
    const wrapper = shallow(
      <UU5.Bricks.Container id={"uuID01"} header="noSpacing={false}" noSpacing={false}>
        <UU5.Bricks.Paragraph id={"uuID02"}/>
      </UU5.Bricks.Container>
    );
    expect(wrapper).toMatchSnapshot();
  });

  it(`UU5.Bricks.Container containt Row and Columns -> should render without crash`, () => {
    const wrapper = shallow(
      <UU5.Bricks.Container id={"uuID01"} header="Container - header">
        <UU5.Bricks.Row id={"uuID02"}>
          <UU5.Bricks.Column id={"uuID03"} colWidth="xs12 m4"><UU5.Bricks.Paragraph/></UU5.Bricks.Column>
          <UU5.Bricks.Column id={"uuID04"} colWidth="xs12 m4"><UU5.Bricks.Paragraph/></UU5.Bricks.Column>
          <UU5.Bricks.Column id={"uuID05"} colWidth="xs12 m4"><UU5.Bricks.Paragraph/></UU5.Bricks.Column>
          </UU5.Bricks.Row>
        </UU5.Bricks.Container>
    );
    expect(wrapper).toMatchSnapshot();
  });

});








