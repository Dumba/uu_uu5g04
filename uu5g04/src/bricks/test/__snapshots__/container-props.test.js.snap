// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UU5.Bricks.Container checkGrammar 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container checkGrammar 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container checkGrammar 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container checkGrammar 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container checkHighlight 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container checkHighlight 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container checkHighlight 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container checkHighlight 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container checkSpaces 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container checkSpaces 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container checkSpaces 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container checkSpaces 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container className 1`] = `
<div
  className="uu5-bricks-container my-classname uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container contains other component UU5.Bricks.Container containt Paragraph and noSpacing is false -> should render without crash 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID01"
  style={Object {}}
  title={null}
>
  <Header
    className={null}
    colorSchema={null}
    content="noSpacing={false}"
    controlled={true}
    disabled={false}
    dynamic={false}
    hidden={false}
    id="uuID01-header"
    ignoreInnerHTML={false}
    key="uuID01-header"
    level={null}
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    style={null}
    tooltip={null}
    underline={false}
  />
  <Paragraph
    className={null}
    content={null}
    controlled={true}
    disabled={false}
    dynamic={false}
    hidden={false}
    id="uuID02"
    ignoreInnerHTML={false}
    key="uuID02"
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    style={null}
    tooltip={null}
  />
</div>
`;

exports[`UU5.Bricks.Container contains other component UU5.Bricks.Container containt Paragraph and noSpacing is true -> should render without crash 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-nospacing"
  id="uuID01"
  style={Object {}}
  title={null}
>
  <Header
    className={null}
    colorSchema={null}
    content="noSpacing={true}"
    controlled={true}
    disabled={false}
    dynamic={false}
    hidden={false}
    id="uuID01-header"
    ignoreInnerHTML={false}
    key="uuID01-header"
    level={null}
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    style={null}
    tooltip={null}
    underline={false}
  />
  <Paragraph
    className={null}
    content={null}
    controlled={true}
    disabled={false}
    dynamic={false}
    hidden={false}
    id="uuID02"
    ignoreInnerHTML={false}
    key="uuID02"
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    style={null}
    tooltip={null}
  />
</div>
`;

exports[`UU5.Bricks.Container contains other component UU5.Bricks.Container containt Row and Columns -> should render without crash 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID01"
  style={Object {}}
  title={null}
>
  <Header
    className={null}
    colorSchema={null}
    content="Container - header"
    controlled={true}
    disabled={false}
    dynamic={false}
    hidden={false}
    id="uuID01-header"
    ignoreInnerHTML={false}
    key="uuID01-header"
    level={null}
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    style={null}
    tooltip={null}
    underline={false}
  />
  <Row
    className={null}
    content={null}
    controlled={true}
    disabled={false}
    display="standard"
    dynamic={false}
    footer={null}
    header={null}
    hidden={false}
    id="uuID02"
    ignoreInnerHTML={false}
    key="uuID02"
    level={null}
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    noSpacing={false}
    pureRender={false}
    ref_={null}
    selected={false}
    style={null}
    tooltip={null}
    underline={false}
  >
    <Column
      className={null}
      colWidth="xs12 m4"
      content={null}
      controlled={true}
      disabled={false}
      dynamic={false}
      footer={null}
      header={null}
      hidden={false}
      id="uuID03"
      ignoreInnerHTML={false}
      level={null}
      mainAttrs={null}
      name={null}
      nestingLevel={null}
      noIndex={false}
      noSpacing={false}
      pureRender={false}
      ref_={null}
      selected={false}
      style={null}
      tooltip={null}
      underline={false}
      width={null}
    >
      <Paragraph
        className={null}
        content={null}
        controlled={true}
        disabled={false}
        dynamic={false}
        hidden={false}
        id={null}
        ignoreInnerHTML={false}
        mainAttrs={null}
        name={null}
        nestingLevel={null}
        noIndex={false}
        pureRender={false}
        ref_={null}
        selected={false}
        style={null}
        tooltip={null}
      />
    </Column>
    <Column
      className={null}
      colWidth="xs12 m4"
      content={null}
      controlled={true}
      disabled={false}
      dynamic={false}
      footer={null}
      header={null}
      hidden={false}
      id="uuID04"
      ignoreInnerHTML={false}
      level={null}
      mainAttrs={null}
      name={null}
      nestingLevel={null}
      noIndex={false}
      noSpacing={false}
      pureRender={false}
      ref_={null}
      selected={false}
      style={null}
      tooltip={null}
      underline={false}
      width={null}
    >
      <Paragraph
        className={null}
        content={null}
        controlled={true}
        disabled={false}
        dynamic={false}
        hidden={false}
        id={null}
        ignoreInnerHTML={false}
        mainAttrs={null}
        name={null}
        nestingLevel={null}
        noIndex={false}
        pureRender={false}
        ref_={null}
        selected={false}
        style={null}
        tooltip={null}
      />
    </Column>
    <Column
      className={null}
      colWidth="xs12 m4"
      content={null}
      controlled={true}
      disabled={false}
      dynamic={false}
      footer={null}
      header={null}
      hidden={false}
      id="uuID05"
      ignoreInnerHTML={false}
      level={null}
      mainAttrs={null}
      name={null}
      nestingLevel={null}
      noIndex={false}
      noSpacing={false}
      pureRender={false}
      ref_={null}
      selected={false}
      style={null}
      tooltip={null}
      underline={false}
      width={null}
    >
      <Paragraph
        className={null}
        content={null}
        controlled={true}
        disabled={false}
        dynamic={false}
        hidden={false}
        id={null}
        ignoreInnerHTML={false}
        mainAttrs={null}
        name={null}
        nestingLevel={null}
        noIndex={false}
        pureRender={false}
        ref_={null}
        selected={false}
        style={null}
        tooltip={null}
      />
    </Column>
  </Row>
</div>
`;

exports[`UU5.Bricks.Container content 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
>
  12345
</div>
`;

exports[`UU5.Bricks.Container content 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
>
  <UU5.Common.TextCorrectorContextConsumer
    key="0"
    text="This is Content from ContentMixin"
  />
</div>
`;

exports[`UU5.Bricks.Container content 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
>
  <UU5.Common.TextCorrectorContextConsumer
    key="0"
    text="This is Content from ContentMixin"
  />
</div>
`;

exports[`UU5.Bricks.Container content 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
>
  12345
</div>
`;

exports[`UU5.Bricks.Container controlled 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container controlled 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container controlled 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container controlled 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container default props 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container disabled 1`] = `
<div
  className="uu5-bricks-container uu5-common-disabled uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
>
  <span
    className="uu5-common-disabled-cover"
    key="disabledCover"
  />
</div>
`;

exports[`UU5.Bricks.Container disabled 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container disabled 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container disabled 4`] = `
<div
  className="uu5-bricks-container uu5-common-disabled uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
>
  <span
    className="uu5-common-disabled-cover"
    key="disabledCover"
  />
</div>
`;

exports[`UU5.Bricks.Container dynamic 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container dynamic 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container dynamic 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container dynamic 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container footer 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
>
  <Footer
    className={null}
    content={
      <div>
        Footer of section
      </div>
    }
    controlled={true}
    disabled={false}
    dynamic={false}
    hidden={false}
    id="uuID-footer"
    ignoreInnerHTML={false}
    key="uuID-footer"
    level={null}
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    style={null}
    tooltip={null}
  />
</div>
`;

exports[`UU5.Bricks.Container header 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
>
  <Header
    className={null}
    colorSchema={null}
    content={
      <div>
        Header of section
      </div>
    }
    controlled={true}
    disabled={false}
    dynamic={false}
    hidden={false}
    id="uuID-header"
    ignoreInnerHTML={false}
    key="uuID-header"
    level={null}
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    style={null}
    tooltip={null}
    underline={false}
  />
</div>
`;

exports[`UU5.Bricks.Container hidden 1`] = `
<div
  className="uu5-bricks-container uu5-common-hidden uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container hidden 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container hidden 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container hidden 4`] = `
<div
  className="uu5-bricks-container uu5-common-hidden uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container id 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="myId"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container ignoreHTML 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container ignoreHTML 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container ignoreHTML 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container ignoreHTML 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 5`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 6`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 7`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 8`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 9`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 10`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 11`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 12`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 13`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 14`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 15`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 16`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 17`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 18`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 19`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 20`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 21`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 22`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 23`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 24`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 25`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 26`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 27`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container level 28`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container mainAttrs 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container name 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  name="myName"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container nestingLevel 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container nestingLevel 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container nestingLevel 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container nestingLevel 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container nestingLevel 5`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container nestingLevel 6`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container nestingLevel 7`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container nestingLevel 8`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container nestingLevel 9`] = `""`;

exports[`UU5.Bricks.Container nestingLevel 10`] = `""`;

exports[`UU5.Bricks.Container nestingLevel 11`] = `""`;

exports[`UU5.Bricks.Container nestingLevel 12`] = `""`;

exports[`UU5.Bricks.Container nestingLevel 13`] = `""`;

exports[`UU5.Bricks.Container nestingLevel 14`] = `""`;

exports[`UU5.Bricks.Container nestingLevel 15`] = `""`;

exports[`UU5.Bricks.Container nestingLevel 16`] = `""`;

exports[`UU5.Bricks.Container nestingLevel 17`] = `""`;

exports[`UU5.Bricks.Container nestingLevel 18`] = `""`;

exports[`UU5.Bricks.Container nestingLevel 19`] = `""`;

exports[`UU5.Bricks.Container nestingLevel 20`] = `""`;

exports[`UU5.Bricks.Container noIndex 1`] = `
<div
  className="uu5-bricks-container uu5-noindex uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container noIndex 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container noIndex 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container noIndex 4`] = `
<div
  className="uu5-bricks-container uu5-noindex uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container noSpacing 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-nospacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container noSpacing 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container noSpacing 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container noSpacing 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-nospacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container pureRender 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container pureRender 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container pureRender 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container pureRender 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container selected 1`] = `
<div
  className="uu5-bricks-container uu5-common-selected uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container selected 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container selected 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container selected 4`] = `
<div
  className="uu5-bricks-container uu5-common-selected uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container style 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={
    Object {
      "borderRadius": "2px",
      "padding": "8px",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Container style 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={
    Object {
      "backgroundColor": "red",
      "color": "blue",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Container style 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={
    Object {
      "backgroundColor": "red",
      "color": "blue",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Container style 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={
    Object {
      "borderRadius": "2px",
      "padding": "8px",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Container textCorrector 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container textCorrector 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container textCorrector 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container textCorrector 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container tooltip 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title="My Tooltip"
/>
`;

exports[`UU5.Bricks.Container underline 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container underline 2`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container underline 3`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Container underline 4`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;
